public/app/features/alerting/unified/components/mute-timings/MuteTimingTimeRange.tsx

```diff
@@ -135,15 +138,18 @@ export const MuteTimingTimeRange = ({ intervalIndex }: Props) => {
           })}
         </>
       </Field>
-      <Button
-        className={styles.addTimeRange}
-        variant="secondary"
-        type="button"
-        icon="plus"
-        onClick={() => addTimeRange({ start_time: '', end_time: '' })}
-      >
-        Add another time range
-      </Button>
+      <Tooltip content={isDisabled ? 'This time interval is disabled' : ''} placement="right-start">
```

Comment:
We might have to use a `ConditionalWrap` here because it looks like it's still rendering a tooltip even with empty content :( 

<img width="296" alt="image" src="https://github.com/grafana/grafana/assets/868844/93cc9560-9a9f-4b7d-bae5-c02f769e6eb0">

Reply:
good catch!